### PR TITLE
Added mocked data to show on profile

### PR DESCRIPTION
**Pull request by hzaw28**

### CODE REVIEW

File: src/views/Home.jsx [hunk 1]
```diff
@@ -20,11 +22,21 @@ export class Home extends Component {
...
+        <div className="page-title">Athletic Tracker App</div>
 
-        <h2><Link to="/status">Status Log</Link></h2>
-        <h2><Link to="/profile">Profile</Link></h2>
-        <button onClick={this.handleOnClick}>Logout</button>
+        <Link to="/status">
+        <Paper 
+          children = {"Status Log"}
+          className = "link-paper"
+        />
```
Comment (solkaz) [inline src/views/Home.jsx]: Can you indent this? Use two spaces, so:
```jsx
<Link to="/status">
  <Paper 
    children={"Status Log"}
    className="link-paper"
  />
</Link>
```

Also, you shouldn't have spaces before/after the `=` when you're assigning props

File: src/views/Profile.jsx [hunk 2]
```diff
@@ -1,29 +1,91 @@
...
+          />
+        </List>
       </div>
     );
   }
 
 }
 
-const mapStateToProps = (state) => {
+const mapStateToProps=(state) => {
```
Comment (solkaz) [inline src/views/Profile.jsx:69]: Here, there should be spaces before/after the `=`

**Review by solkaz — CHANGES_REQUESTED** [review]
I'm going to add Eslint so we have consistent code style. Brb

**Comment by solkaz** [comment]
You'll want to merge the master branch into your feature branch as I've added eslint so there aren't code styling issues

File: src/views/Home.jsx (hunk 1 above)
Comment (hzaw28) [reply src/views/Home.jsx]: I used ms vs-code, it was not intentional.

Comment (solkaz) [reply src/views/Home.jsx]: You should be able to set your tabs to be spaces (soft-tabs). That would be preferable

File: src/views/Profile.jsx (hunk 2 above)
Comment (heinhtetzaw95) [inline src/views/Profile.jsx]: [https://scontent-ort2-1.xx.fbcdn.net/v/t35.0-12/17622574_1474585682565872_950110749_o.png?oh=b2c079b68e943f838a1e9f0d58da2f30&oe=58DF95C4](url)

**Review by solkaz — APPROVED** [review]

**Pull request merged**